recipes/rapidcheck/all/conanfile.py

```diff
@@ -62,10 +74,15 @@ def _configure_cmake(self):
         self._cmake.definitions["RC_ENABLE_RTTI"] = self.options.enable_rtti
         self._cmake.definitions["RC_ENABLE_TESTS"] = False
         self._cmake.definitions["RC_ENABLE_EXAMPLES"] = False
+        self._cmake.definitions["RC_ENABLE_CATCH"] = self.options.enable_catch
+        self._cmake.definitions["RC_ENABLE_GMOCK"] = self.options.enable_gmock
+        self._cmake.definitions["RC_ENABLE_GTEST"] = self.options.enable_gtest
         self._cmake.configure(build_folder=self._build_subfolder)
         return self._cmake
 
     def build(self):
```

Comment:
```suggestion
    def build(self):
        if self.options.enable_gmock and not self.deps_cpp_info["gtest"].build_gmock:
            raise ConanInvalidConfiguration("The option `rapidcheck:enable_gmock` requires gtest:build_gmock=True`")
```

The gmock is optional for Gtest. By default it's enabled, but still anyone can use it disabled. It should be checked in build(), because deps_cpp_info will be populated only before build method.